test/clt-tests/buddy/test-fuzzy-search-negative.rec

```diff
@@ -17,16 +17,44 @@ ERROR 1064 (42000) at line 1: P01: syntax error, unexpected '-' near '-1'
 ––– input –––
 mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('SMITH') OPTION fuzzy='0';"
 ––– output –––
-ERROR 1064 (42000) at line 1: unknown option 'fuzzy'
+ERROR 1064 (42000) at line 1: Fuzzy plugin requires min_infix_len to be set
 ––– input –––
 mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('SMYTH') OPTION fuzzy='2';"
 ––– output –––
-ERROR 1064 (42000) at line 1: unknown option 'fuzzy'
+ERROR 1064 (42000) at line 1: Fuzzy plugin requires min_infix_len to be set
 ––– input –––
 mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('john') OPTION fuzzy=a;"
 ––– output –––
-ERROR 1064 (42000) at line 1: unknown option 'fuzzy'
+ERROR 1064 (42000) at line 1: Fuzzy plugin requires min_infix_len to be set
 ––– input –––
 mysql -h0 -P9306 -e "SELECT * FROM name WHERE MATCH('jane') OPTION fuzzy=@;"
 ––– output –––
-ERROR 1064 (42000) at line 1: unknown option 'fuzzy'
+ERROR 1064 (42000) at line 1: Fuzzy plugin requires min_infix_len to be set
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('j', 'name', 'us' AS layouts);"
+––– output –––
+ERROR 1064 (42000) at line 1: Autocomplete plugin requires min_infix_len to be set
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE(123, 'name', 'us' AS layouts);"
+––– output –––
+ERROR 1064 (42000) at line 1: Something went wrong
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('', 'name', 'us' AS layouts);"
+––– output –––
+ERROR 1064 (42000) at line 1: Something went wrong
+––– input –––
+mysql -h0 -P9306 -e "CALL AUTOCOMPLETE('@#$', 'name', 'us' AS layouts);"
+––– output –––
+ERROR 1064 (42000) at line 1: Autocomplete plugin requires min_infix_len to be set
```

Comment:
CALL AUTOCOMPLETE requires min_infix_len to be set in the table settings.

Reply:
Fixed in a separate PR